src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -53,7 +53,6 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
 
     // type name for this object, used as the ES-type
     const esType = config.type;
-    this.index = kbnIndex;
```

Comment:
This is why I love getters so much, easier to find uses but mostly that unexpected accesses that are broken by changes/removal are much louder